.circleci/config.yml

```diff
@@ -95,11 +95,14 @@ commands:
             make -j$(nproc) && make install"
   test_flashlight_inside_nvidia_docker:
     parameters:
+      test_name:
+        type: string
+        default: ""
```

Comment:
nit: if you leave out the default, it'll be required, which might be what we want

Reply:
My intention was to not have it be required actually. Let me know if you think it should be, but it's only purpose is to add some information to the circle CI UI, so not really required functionally.